src/Core/Components/DesignSystemProvider/FluentDesignTheme.razor.cs

```diff
@@ -156,13 +162,14 @@ protected async override Task OnAfterRenderAsync(bool firstRender)
             Module ??= await JSRuntime.InvokeAsync<IJSObjectReference>("import", JAVASCRIPT_FILE);
             _dotNetHelper = DotNetObjectReference.Create(this);
 
-            _direction = await Module.InvokeAsync<string?>("GetDirection");
+            if (!Enum.TryParse(await Module.InvokeAsync<string>("GetDirection"), true, out _direction))
+                _direction = LocalizationDirection.ltr;
 
-            var themeJSON = await Module.InvokeAsync<string>("addThemeChangeEvent", _dotNetHelper, Id);
-            var theme = themeJSON == null ? null : JsonSerializer.Deserialize<DataLocalStorage>(themeJSON, JSON_OPTIONS);
+            string? themeJSON = await Module.InvokeAsync<string>("addThemeChangeEvent", _dotNetHelper, Id);
```

Comment:
A reason why you changed `var` to `string?` and `DataLocalStorage?` ?

Reply:
I don't like `var` :).
In VS it displays like this (for me based on my settings, only changed it for the first line):
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/a06d1da2-5970-4582-b694-5cc7f6668253)

If the type is already known an shown, then why not us it? VS actually does the substitution for you if you double check on the type.

Reply:
It shouldn't depend on preferences (like or dislike), but on naming convention 😀
Here's the one we use in many MS projects: https://dvoituron.com/naming-conventions/rules/

![image](https://github.com/microsoft/fluentui-blazor/assets/8350694/21b411eb-b8c0-44af-b987-cd20cbef39cc)

The code is clearer / aligned with `var` and it's more " scalable " when the objects (right) evolve.
But it's only a naming convention.

```csharp
var themeJSON = ...
var theme = ...
```